packages/ceramic-core/src/ceramic.ts

```diff
@@ -48,6 +50,8 @@ export interface CeramicConfig {
   gateway?: boolean;
 
   topic?: string;
+
+  [index: string]: any; // allow arbitrary properties
```

Comment:
Maybe rather than allowing arbitrary config props, we could define the supported interfaces?
For example, we could have something like `anchorService?: EthereumAnchorServiceConfig | InMemoryAnchorServiceConfig`?

Reply:
Yep, makes sense 👍  I think this is acceptable for now since anchor service(s) will be refactored as part of a separate story soon.